servicetalk-http-netty/src/main/java/io/servicetalk/http/netty/ProxyConnectChannelSingle.java

```diff
@@ -193,7 +195,7 @@ public void userEventTriggered(final ChannelHandlerContext ctx, final Object evt
             connectObserver.proxyConnectComplete(response);
             ctx.pipeline().remove(this);
             final Channel channel = ctx.channel();
-            LOGGER.debug("{} Received successful response from proxy on CONNECT {}", channel, connectAddress);
+            LOGGER.debug("{} Received successful response from proxy on CONNECT {}", channel, proxyConfig.address());
```

Comment:
Does it make sense to include the name of the headers initializer? For really adversarially defined initializers I can see that leading to the case of logging headers so maybe it's not a good idea.

Reply:
No, here we care only for what address we failed or were successful. The rest of the context either should not matter or users may infer it from their codebase